src/Symfony/Component/Form/Tests/ChoiceList/ArrayChoiceListTest.php

```diff
@@ -18,13 +18,9 @@
  */
 class ArrayChoiceListTest extends AbstractChoiceListTest
 {
-    private $object;
-
     protected function setUp()
     {
         parent::setUp();
-
-        $this->object = new \stdClass();
```

Comment:
this was broken because it was initialized after `getChoices` is called. so `$this->object` was actually null.